README.md

```diff
@@ -103,11 +104,22 @@ if (project.hasProperty("cfLocal")) {
     compileOnly files(cfHome + "/checker/dist/checker-qual.jar")
     testCompileOnly files(cfHome + "/checker/dist/checker-qual.jar")
     checkerFramework files(cfHome + "/checker/dist/checker.jar")
+    // only needed for JDK 8
     checkerFrameworkAnnotatedJDK files(cfHome + "/checker/dist/jdk8.jar")
   }
 }
 ```
 
+### JDK 8 vs JDK 9+
+
+The plugin attempts to automatically configure the Checker Framework on both Java 8 and Java 9+ JVMs,
+following the [best practices in the Checker Framework manual](https://checkerframework.org/manual/#javac).
+In particular:
+* If both the JVM and target versions are 8, it applies the Java 8 annotated JDK.
+* If the JVM version is 9+ and the target version is 8 (and the Checker Framework
+version is >= 2.11.0), use the Error Prone javac compiler.
+* If the JVM version is 9+, use the `--add-opens` option to `javac`.
```

Comment:
This is done even if the CF version is < 2.11?

Reply:
Yes, but in that case it doesn't really matter, because the CF won't run.